gradle.properties

```diff
@@ -1,5 +1,6 @@
 VERSION_NAME=6.1.0
 VERSION_CODE=1
-ANDROID_BUILD_MIN_SDK_VERSION=12
+ANDROID_BUILD_MIN_SDK_VERSION=24
 ANDROID_BUILD_TARGET_SDK_VERSION=27
```

Comment:
Maybe would be nice if this could be higher, like android 34, or 33?  Though it may not make a difference.  I don't believe compiling our flutter app was needing android sdk 27 - so I assume the reason flutter dependencies can end up requiring older SDK versions is if flutter isn't relying on precompiled versions of flutter package's android code? In this case it's using a hosted pre-compiled version from maven/jitpack so it we don't need the compile sdk

Reply:
Changing target SDK version means that the library must behave as expected in the target environment.
I haven't checked the code for any breaking change targeting 34.
I am not confident on touching this version number if there is no real benefit to it.